### PR TITLE
Ask users for `go list ...` output

### PR DESCRIPTION
**Pull request by lynncyrin**
I noticed via these two issues https://github.com/urfave/cli/issues/964, https://github.com/urfave/cli/issues/957 that people are liking to just say "v1" or "v2" which sometimes won't be enough detail. Other times, it won't be correct!

So I've changed the issue template to instead prompt for `go list -u -m all`, which looks like this

```
❯ go list -u -m all
go: finding github.com/cpuguy83/go-md2man/v2 v2.0.0
go: finding gopkg.in/check.v1 latest
go: finding gopkg.in/yaml.v2 v2.2.7
github.com/urfave/cli/v2
github.com/BurntSushi/toml v0.3.1
github.com/cpuguy83/go-md2man/v2 v2.0.0-20190314233015-f79a8a8ca69d [v2.0.0]
github.com/pmezard/go-difflib v1.0.0
github.com/russross/blackfriday/v2 v2.0.1
github.com/shurcooL/sanitized_anchor_name v1.0.0
gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405 [v1.0.0-20190902080502-41f04d3bba15]
gopkg.in/yaml.v2 v2.2.2 [v2.2.7]
```

😍  beautiful! so much information!

### CODE REVIEW

**Comment by lynncyrin** [comment]
> maybe we should add a little script where the output can be pasted in the issue

!!! I'll think on this 🤔

**Comment by lynncyrin** [comment]
ping @asahasrabuddhe & @AudriusButkevicius

File: .github/ISSUE_TEMPLATE/v1-bug-report.md [hunk 1]
```diff
@@ -59,3 +55,9 @@ We'd love to have more contributors on this project! If the fix for this bug is
 ```
 # paste `go env` output in here
 ```
+
+## Run `go list -u -m all` and paste its output here
+
+```
+# paste `go list -u -m all` output in here
```
Comment (AudriusButkevicius) [inline .github/ISSUE_TEMPLATE/v1-bug-report.md]:
```
~ $ go list -u -m all
warning: pattern "all" matched no module dependencies
```

What is this supposed to do?

Comment (AudriusButkevicius) [reply .github/ISSUE_TEMPLATE/v1-bug-report.md]: Ok, we have to be more explicit where this needs to be run. This also can be pretty big, are we sure we want everything?
What are we trying to answer here?

Comment (lynncyrin) [reply .github/ISSUE_TEMPLATE/v1-bug-report.md]: > What is this supposed to do?

I added that into my pull request description

```
❯ go list -u -m all
go: finding github.com/cpuguy83/go-md2man/v2 v2.0.0
go: finding gopkg.in/check.v1 latest
go: finding gopkg.in/yaml.v2 v2.2.7
github.com/urfave/cli/v2
github.com/BurntSushi/toml v0.3.1
github.com/cpuguy83/go-md2man/v2 v2.0.0-20190314233015-f79a8a8ca69d [v2.0.0]
github.com/pmezard/go-difflib v1.0.0
github.com/russross/blackfriday/v2 v2.0.1
github.com/shurcooL/sanitized_anchor_name v1.0.0
gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405 [v1.0.0-20190902080502-41f04d3bba15]
gopkg.in/yaml.v2 v2.2.2 [v2.2.7]
```

did you run it inside of a golang github repo? what go version are you using?

> This also can be pretty big, are we sure we want everything?

Yes

> What are we trying to answer here?

The versions of your dependencies. Most critically the version of urfave/cli

Comment (asahasrabuddhe) [reply .github/ISSUE_TEMPLATE/v1-bug-report.md]: I think we can pipe this to ```grep urfave/cli``` to just focus on the version of our package.

**Comment by lynncyrin** [comment]
🕙 closing due to age _(and my attention span)_

**Pull request closed without merging**